### ISSUE TITLE
[Deployment UX] SSL Certificate Provisioning

### ISSUE THREAD

**Issue by mikeshultz** [OriginProtocol/dshop#683]
Right now, when creating a new CDN endpoint, [GCP can take up to 60 minutes](https://cloud.google.com/load-balancing/docs/ssl-certificates/troubleshooting?&_ga=2.165840060.-360165820.1597679396#certificate-managed-status) to issue and deploy the cert.

In practice, I have yet to see it take that long(usually minutes) but it may still be confusing to users. In the mean time, before provisioning, Chrome returns the ERR_SSL_VERSION_OR_CIPHER_MISMATCH error.

This gets a little more confusing when switching from a CNAMEd IPFS deployed shop to a CDN.

I think the deployment module should probably wait on this until a successful SSL connection can be made. Though I'm not sure if a 5-15minute spinner is really ideal here.

Ref: #669

**Comment by mikeshultz** [comment]
CC @shahthepro @nick @franckc 

Wondering if in the very short-term (can throw this out today) we can manage expectations with a message on one of these modals:

![publishing](https://user-images.githubusercontent.com/1048089/97039573-e3863d00-1529-11eb-9644-2fdfac7761f3.png)

![finished](https://user-images.githubusercontent.com/1048089/97039590-e8e38780-1529-11eb-9b8c-b6c146a9993c.png)

I'm thinking something along the lines of this?

> We're in the process of deploying your shop to a global content distribution network and the Interplanetary Flie System (IPFS).  It may take up to 15 minutes for all changes to take effect.  Please be patient.

**Comment by franckc** [comment]
@mikeshultz Yes I think it's a great idea to add that text. I would suggest adding it on the last modal "Your changes have been successfull published".

**Comment by mikeshultz** [comment]
Ref #695

**Comment by micahalcorn** [comment]
@mikeshultz is this something you're working on or finished? If not, shall we move it to the backlog?

**Comment by mikeshultz** [comment]
Again, in progress but incomplete.  Backlog if you want.
